cloudplatform/cloudplatform-connectivity/src/main/java/com/sap/cloud/sdk/cloudplatform/connectivity/BtpServiceOptions.java

```diff
@@ -142,29 +142,21 @@ public static OptionsEnhancer<?> withTargetUri( @Nonnull final URI targetUri )
         }
 
         /**
-         * Creates an {@link OptionsEnhancer} that instructs the destination to use mTLS authentication only.
-         * <p>
-         * In the regular OAuth2 case, setting this option will skip the IAS token flow.
-         * <p>
-         * <b>Hint:</b> This option is <b>mutually exclusive</b> with {@link #withApplicationName(String)} and
-         * {@link #withConsumerClient(String, String)}.
-         * <p>
-         * <b>Caution:</b> This option cannot be combined with {@link OnBehalfOf#NAMED_USER_CURRENT_TENANT}.
+         * Creates an instance of {@link NoTokenForTechnicalProviderAccess}.
          *
-         * @return An instance of {@link OptionsEnhancer} that will lead to mTLS authentication only.
+         * @return A new {@link NoTokenForTechnicalProviderAccess} instance.
          */
         @Nonnull
-        public static OptionsEnhancer<?> withMTLSAuthenticationOnly()
+        public static OptionsEnhancer<?> withMutualTlsForTechnicalProviderAuthenticationOnly()
```

Comment:
I feel that even with such a lengthy name it isn't super clear. One could also read this as "only a technical user of the provider account needs to do mTLS" or "only the authentication part for the provider case (not the OAuth flow) requires mTLS"... 

How about:
```suggestion
        public static OptionsEnhancer<?> withoutTokenForBehalfTechnicalUserProvider()
```

or something similar?

Reply:
I feel the code would become a bit easer to read if we align the naming in the different places a bit more. Of course, the semantic is ever so slightly different in the different places, but still 😄

Reply:
mhmm... What do you think about `withoutTokenForTechnicalProviderUser`?

Reply:
sounds good 👍🏻